README.md

```diff
@@ -1,92 +1,180 @@
-# CXX API for CycloneDDS
+# C++ binding for Eclipse Cyclone DDS
 
-This project provides a CXX API for [CycloneDDS](https://github.com/eclipse-cyclonedds/cyclonedds/)
+An implementation of the [ISO/IEC C++ PSM][1], or simply put, a C++ binding
+for [Eclipse Cyclone DDS][2]. Cyclone DDS is developed completely in the open
+as an Eclipse IoT project (see [eclipse-cyclone-dds][3]) with a growing list
+of [adopters][4]i (if you're one of them, please ad your [logo][5]). It is a
+tier-1 middleware for the Robot Operating System [ROS 2][6].
 
-[![Build Status](https://dev.azure.com/thijssassen/CXX-API/_apis/build/status/ThijsSassen.cdds-cxx?branchName=master)](https://dev.azure.com/thijssassen/CXX-API/_build/latest?definitionId=4&branchName=master)
+[1]: https://www.omg.org/spec/DDS-PSM-Cxx/
+[2]: https://github.com/eclipse-cyclonedds/cyclonedds/
+[3]: https://projects.eclipse.org/projects/iot.cyclonedds
+[4]: https://iot.eclipse.org/adopters/?#iot.cyclonedds
+[5]: https://github.com/EclipseFdn/iot.eclipse.org/issues/new?template=adopter_request.md
+[6]: https://index.ros.org/doc/ros2/
 
-## Requirements for the CXX API
 
-In order to build the Idl compiler you need a Linux, Mac or Windows 10 machine with the following
-installed on your host:
+# Getting Started
 
-  * Git
-  * [Conan](https://conan.io/), this is needed to install build dependencies
-  * [CMake](https://cmake.org/download/), version 3.7 or later.  (Version 3.6 should work but you
-    will have to edit the ``cmake_minimum_required`` version.)
-  * [CycloneDDS](https://github.com/eclipse-cyclonedds/cyclonedds/)
-  * [CXX Idl compiler](https://github.com/ADLINK-IST/idlpp-cxx/)
+## Building Eclipse Cyclone DDS CXX
 
+In order to build the C++ binding for Cyclone DDS you need a Linux, Mac or
+Windows 10 machine (or, with some caveats, a \*BSD, OpenIndiana one) with the
+following installed on your host:
 
-## Building
+ * C and C++ compilers (most commonly GCC on Linux, Visual Studio on Windows,
+   Xcode on macOS);
+ * [Git](https://git-scm.com/) version control system;
+ * [CMake](https://cmake.org/download/), version 3.7 or later;
+ * [Eclipse Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds/)
+ * [CXX Idl compiler](https://github.com/ADLINK-IST/idlpp-cxx/)
```

Comment:
I think it would be advisable to add a note that this is a temporary situation and that there will soon be a new IDL compiler in Cyclone that will also support C++